source/includes/quick-start/deleteOne.go

```diff
@@ -0,0 +1,41 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/mongo"
+	"go.mongodb.org/mongo-driver/mongo/options"
+)
+
+// Replace the uri string with your MongoDB deployment's connection string.
+const uri = "mongodb+srv://<username>:<password>@<cluster-address>/test?w=majority"
+	
+func main() {
+
+	ctx := context.TODO()
+	client, err := mongo.Connect(ctx, options.Client().ApplyURI(uri))
+	if err != nil {
+		panic(err)
+	}
+	defer func() {
+		if err = client.Disconnect(ctx); err != nil {
+			panic(err)
+		}
+	}()
+
+	// begin deleteOne
+	coll := client.Database("sample_mflix").Collection("movies")
+
+	result, err := coll.DeleteOne(ctx, bson.D{{"title", "The Room"}})
+	// end deleteOne
+
+	if err != nil {
+		log.Panic(err)
+	}
+
+	fmt.Printf("%v document was deleted.\n", result.DeletedCount)
```

Comment:
Question: Would it make sense to provide a reset function for examples that write to the database? Not sure, just curious. Could see arguments going either way.